node-require/calculate.js

```diff
@@ -0,0 +1,18 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+const arg = process.argv;
```

Comment:
It's more common to name this `args` (plural) since there are more than one arguments.